File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
a, b, c = map(int,input().split())

if (a==b==c):
    n = 10000 + a*1000
elif (a==b):
    n = 1000 + a*100
elif (b==c):
    n = 1000 + b*100
elif (c==a):
    n = 1000 + c*100
else:
    n = max(a,b,c)*100
    
print(n)

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.kt
import java.util.*
import java.util.Collections

fun main() = with(Scanner(System.`in`)) {
    var (a,b,c) = readln().split(' ').map { it.toInt() }
    val list = listOf(a,b,c)
    var n : Int

    if(a==b && b==c){
        n = 10000 + a*1000
    }else if(a==b){
        n=1000+a*100
    }else if(b==c){
        n = 1000 + b*100
    }else if(c==a){
        n = 1000 + c*100
    }else{
        n=Collections.max(list)*100
    }

    print(n)
}